core/src/Kokkos_ExecPolicy.hpp

```diff
@@ -40,6 +40,12 @@ struct ChunkSize {
 #endif
 };
 
+namespace Impl {
+// Tag to be given when calling a policy constructor that
+// updates an existing policy.
```

Comment:
(Trying to improve the comment)

Private tag that can be used to make a copy of another execution policy and swap the underlying execution space instance (needed for Kokkos::Graph)

Reply:
OK, will do :+1: